pinot-controller/src/main/resources/app/utils/PinotMethodUtils.ts

```diff
@@ -301,13 +303,13 @@ const getQueryResults = (params, url, checkedOptions) => {
       },
       queryStats: {
         columns: columnStats,
-        records: [[data.timeUsedMs, data.numDocsScanned, data.totalDocs, data.numServersQueried, data.numServersResponded,
```

Comment:
`data = JSONbig.parse(data)` was only added recently, so I'm not really sure how this was working before. But now this just reuses the `queryResponse` object

Reply:
Yes it was added recently and before that we used to directly use `data` from `return getQueryResult(params, url).then(({ data }) => {`